packages/eip1193/src/index.spec.ts

```diff
@@ -1,8 +1,7 @@
 import { Eip1193Bridge } from '@ethersproject/experimental'
 import { Web3Provider } from '@ethersproject/providers'
 import { createWeb3ReactStoreAndActions } from '@web3-react/store'
-import type { Actions, ProviderRpcError, RequestArguments, Web3ReactStore } from '@web3-react/types'
-import { EventEmitter } from 'node:events'
+import type { Actions, Web3ReactStore } from '@web3-react/types'
```

Comment:
Should `MockProviderRpcError` extend `ProviderRpcError`?

Reply:
What about `RequestArguments`? Do you know why that was there?

Reply:
this shouldn't be a blocker so I'm still going to approve, but it seems like it may have been unintended to extend from `Error` instead of `ProviderRpcError`.

Reply:
I'll update the type definitions to make them more explicit

Reply:
I am going to work on the types in a follow-up PR, already making a TODO in my next PR.

Reply:
> Should MockProviderRpcError extend ProviderRpcError?

Can't extend, but should `implement` instead. Added to make sure it follows the type.

Reply:
> this shouldn't be a blocker so I'm still going to approve, but it seems like it may have been unintended to extend from Error instead of ProviderRpcError.

We can't extend from `ProviderRpcError` because it's not a class, it's a TypeScript interface. We can only `implement` it, which we already do. I will submit a follow-up PR now.